lib/coarse_op.cuh

```diff
@@ -791,7 +791,10 @@ namespace quda {
 
       if (type == COMPUTE_UV) {
         strcat(Aux, ",computeUV");
-        if (use_mma) strcat(Aux, ",mma");
+        if (use_mma) {
+          strcat(Aux, ",mma");
+          strcat(Aux, mma::mg_mma_dispatch_t<Float>::type::get_type_name().c_str());
```

Comment:
Should this extra bit also be added to the `COMPUTE_VUV` tunekey?

Reply:
It has been added to VUV as well if you check the latest of this branch.